acid/features/history/tests/conftest.py

```diff
@@ -11,13 +11,30 @@
 @pytest.fixture
 def zuul_builds(mocker):
     mocker.patch('pony.orm.core.commit')
+
     return [make_build(build_number=104, buildset_id=5010),
             make_build(build_number=104, buildset_id=5010)]
 
 
+@pytest.fixture
+def zuul_build(mocker):
+    mocker.patch('pony.orm.core.commit')
+
+    def _make_build(build_number=104, buildset_id=5010, **kwargs):
```

Comment:
This function looks the same as https://github.com/codilime/acid/pull/35/files#diff-281c57eb36c36bda9704ad2b9d20b66fR57
Reuse instead of duplicating.